tests/dialects/test_builtin.py

```diff
@@ -73,13 +73,14 @@ def test_FloatType_formats():
 
 
 def test_IntegerType_formats():
-    with pytest.raises(NotImplementedError):
-        IntegerType(2).format
     assert IntegerType(1).format == "<b"
+    assert IntegerType(2).format == "<b"
     assert IntegerType(8).format == "<b"
     assert IntegerType(16).format == "<h"
     assert IntegerType(32).format == "<i"
     assert IntegerType(64).format == "<q"
+    with pytest.raises(NotImplementedError):
+        IntegerType(65).format
```

Comment:
Let's add these sanity checks?
```suggestion
    assert IntegerType(1).format == "<b"
    assert IntegerType(2).format == "<b"
    assert IntegerType(8).format == "<b"
    assert IntegerType(15).format == "<h"
    assert IntegerType(16).format == "<h"
    assert IntegerType(31).format == "<i"
    assert IntegerType(32).format == "<i"
    assert IntegerType(63).format == "<q"
    assert IntegerType(64).format == "<q"
    with pytest.raises(NotImplementedError):
        IntegerType(65).format
```